pypeit/spectrographs/spectrograph.py

```diff
@@ -1441,14 +1441,33 @@ def check_disperser(self):
     @property
     def order_spat_pos(self):
         """
+        This is for fixed-format echelle spectrographs (e.g. X-Shooter)
+
         Return the expected spatial position of each echelle order.
+
+        Measured 1/2 way up the chip (spectral)
+        and normalized units (0-1)
+
+
+        Args:
+
+        Returns:
+            `numpy.ndarray`_: An array with values. 
+                The length of the provided array much 
+                match self.norders
```

Comment:
A few things:

- The first line of the docstring should be a brief description of what the function's purpose, followed by a longer description and caveats.
- If there are no arguments, there shouldn't be an `Args:` section.
- It might not complain when the docs are built, but my guess is that the indentation in the return statement isn't going to display correctly.  Unless you're providing a list of items, return descriptions in Google format should all have the same indentation (i.e., get rid of the indentation in lines 2 & 3 of the return description).

Reply:
all done